source/quickinstallationguide/qig.rst

```diff
@@ -433,6 +433,13 @@ Install Python MySQL connector from the MySQL community repository (which we've
 Please note that the previously required ``mysql-connector-java`` library is now bundled with CloudStack
 Management server and is no more required to be installed separately.
 
```

Comment:
Seems the changes here, are not related to disk provisioning types. Please check.

Reply:
This was a conflict that I had to resolve

Reply:
can you rebase and resolve properly. this changes shouldn't come up as changes in this PR.